src/js/stores/MesosStateStore.js

```diff
@@ -288,7 +297,12 @@ class MesosStateStore extends GetSetBaseStore {
     // Marathon tasks. This will give you a list of framework tasks including
     // the scheduler tasks or a list of Marathon application tasks.
     return frameworks.reduce(function(serviceTasks, framework) {
-      const { tasks = [], completed_tasks = {}, name } = framework;
+      const {
+        tasks = [],
+        completed_tasks = {},
```

Comment:
I am going to find out why it is an object I speculated as well but then figured that there was some reason. I didn't changed this tho'.